cmake/OatppWebSocketExternalProject.cmake

```diff
@@ -0,0 +1,21 @@
+#[[ CMake guard. ]]
+if(TARGET OatppExternalProject)
+    return()
+endif()
+
+include(ExternalProject)
+
```

Comment:
I am assuming you need Oatpp built and installed first. If that is the case, then this file should include the OatppExternalProject.cmake file here. You can refer to CeresSolverExternalProject.cmake for an example.

Reply:
You will also need to add an dependency between OatppWebsockerExternalProject and OatppExternalProject so that cmake know to build one before the other in a parallel make.